src/porepy/numerics/fv/fv_elliptic.py

```diff
@@ -344,3 +347,429 @@ def enforce_neumann_int_bound(
         """
         # Operation is void for finite volume methods
         pass
+
+
+class FVVectorElliptic(pp.numerics.mixed_dim.solver.Solver):
```

Comment:
Nothing should inherit from the old solver anymore - the whole concept should be purged. This PR is not the place for that, though.